<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>taskDataMenu</class>
 <widget class="QDialog" name="taskDataMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="addTaskButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add task</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>21</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selected workers</string>
   </property>
  </widget>
  <widget class="QLabel" name="workersLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addWorkerButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add worker</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeWorkerButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove worker</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>45</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Hours</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="endHoursSpinBox">
      <property name="maximum">
       <number>23</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Minutes</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="endMinutesSpinBox">
      <property name="maximum">
       <number>59</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Beginning time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>End time</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>74</width>
     <height>228</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Workers</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="workersComboBox"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Task type</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="taskTypeComboBox"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Time</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="timeComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>45</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Hours</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="beginHoursSpinBox">
      <property name="maximum">
       <number>23</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Minutes</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="beginMinutesSpinBox">
      <property name="maximum">
       <number>59</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
